### ISSUE TITLE
Cubieboard Hangs

### ISSUE THREAD

**Issue by McBodik** [cubieplayer/Cubian#70]
Hi again. I connect to my Cubieboard via ssh. After some uptime connecting break. First I thought it's network problem. But I had connected board to monitor and there wasn't any signal. After reboot Cubian start without problems and work fine some time to the next hangs. 

It's system or  board problem?

**Comment by cubieplayer** [comment]
May be your cubieboard needs a cooler

**Comment by zerodamage** [comment]
Ive had random crashes as well, a simple cooler has helped somewhat. You can also underclock if performance is not critical. Also ensure the SD card is good quality.

You can check system logs too see if its triggered by kernel panic or similar

**Comment by s-kostyuk** [comment]
Similar problem for me and it's overheating. How to reproduce: start any resource-intensive process like nandinstall or memtester on Cubieboard with temperature approximately 25°C in room and wait several minutes. As result: kernel panic, kernel oops or simple reboot without any messages. But with good cooler all is OK.

Sorry for my English.

**Comment by siddhy** [comment]
Hi,

I have also experienced a hanging issue with my cubieboard 1. Actually it hanged after a few minutes, generally during a package installation, through ssh: during the installation, apt stopped to display anything, and I weren't able to ping the cubie.

This issue happened after I added a USB cable extension between the main supply and the Cubie. So I switched back to direct connection through USB to jack cable, and I have no more issue. I have not checked that it is effectively a cable issue, but maybe you are experiencing the same issue ? Either your power supply or your cable is not well suited ?
